src/patterns/molecules/notification-bar/notification-bar.hbs

```diff
@@ -0,0 +1,90 @@
+---
+name: Notification bar
+description: A notification bar provides contextual feedback messages
+notes: |
+    - Notification bars have a range of different colors
+    - `mzp-t-success`
+    - `mzp-t-warning`
+    - `mzp-t-error`
+    - `mzp-t-click`
+    - An optional dismiss button
+    - `mzp-has-dismiss`
+    - And optional stickiness
+    - `mzp-is-sticky`
```

Comment:
Suggest adding documentation here as to what options the JavaScript API comes with?